src/main/java/net/minecraftforge/fml/common/network/FMLOutboundHandler.java

```diff
@@ -43,9 +44,10 @@
 import javax.annotation.Nullable;
 
 public class FMLOutboundHandler extends ChannelOutboundHandlerAdapter {
-    public static final AttributeKey<OutboundTarget> FML_MESSAGETARGET = AttributeKey.valueOf("fml:outboundTarget");
+    public static final AttributeKey<IOutboundTarget> FML_MESSAGETARGET = AttributeKey.valueOf("fml:outboundTarget");
```

Comment:
Unfortunately Forge's policy is to take no chances on this kind of thing, since it's public we have to assume someone is using it. There are snapshots for 1.12 already it's probably not that far off, so I think you should leave this as `AttributeKey<OutboundTarget>` until then.